### PR TITLE
Added broadcastMessage methods to PlayerExecutor

### PR DESCRIPTION
**Pull request by einrobin**

### CODE REVIEW

**Comment by GiantTreeLP** [comment]
Can you add a broadcast method that takes a `BaseComponent[]` as well?

**Comment by derklaro** [comment]
In my opinion you can remove it out of the CorePlayerExecutor because there will never come an implementation.

**Comment by einrobin** [comment]
Why should I remove that out of the CorePlayerExecutor? I would use that broadcastMessage methods for my systems.

**Comment by derklaro** [comment]
The CorePlayerExecutor and the PlayerExecutor are unused classes.
It's not needed to add some more methods, some are empty, when all other methods don't work.

It's a null instance

**Comment by einrobin** [comment]
The CorePlayerExecutor ist used, of course. And I think I will implement those methods that are not implemented in it yet.

**Comment by derklaro** [comment]
Yes it is used, but all other methods are not in use.  It don't make any sence to impent only 4 methos.

**Comment by einrobin** [comment]
But I've already said that I will implement those unimplemented methods soon.

File: cloudnet-core/src/main/java/de/dytanic/cloudnetcore/api/event/player/UpdatePlayerEvent.java [hunk 1]
```diff
@@ -15,11 +15,13 @@
 @Getter
```
Comment (GiantTreeLP) [inline cloudnet-core/src/main/java/de/dytanic/cloudnetcore/api/event/player/UpdatePlayerEvent.java:15]: This file has nothing to do with the pull request

File: cloudnet-core/src/main/java/de/dytanic/cloudnetcore/database/PlayerDatabase.java [hunk 2]
```diff
@@ -12,6 +12,8 @@
 import de.dytanic.cloudnet.lib.player.PlayerConnection;
```
Comment (GiantTreeLP) [inline cloudnet-core/src/main/java/de/dytanic/cloudnetcore/database/PlayerDatabase.java:12]: This file has nothing to do with the pull request

File: cloudnet-core/src/main/java/de/dytanic/cloudnetcore/network/NetworkManager.java [hunk 3]
```diff
@@ -269,8 +269,9 @@ public void handleCommandExecute(PlayerCommandExecution playerCommandExecutor)
```
Comment (GiantTreeLP) [inline cloudnet-core/src/main/java/de/dytanic/cloudnetcore/network/NetworkManager.java:269]: This file has nothing to do with the pull request

File: cloudnet-lib/src/main/java/net/md_5/bungee/api/ChatColor.java [hunk 4]
```diff
@@ -0,0 +1,192 @@
+package net.md_5.bungee.api;
```
Comment (GiantTreeLP) [inline cloudnet-lib/src/main/java/net/md_5/bungee/api/ChatColor.java:1]: Please mark where this file has come from and when in a comment above the package

File: cloudnet-lib/src/main/java/net/md_5/bungee/api/ChatMessageType.java [hunk 5]
```diff
@@ -0,0 +1,12 @@
+package net.md_5.bungee.api;
```
Comment (GiantTreeLP) [inline cloudnet-lib/src/main/java/net/md_5/bungee/api/ChatMessageType.java:1]: Please mark where this file has come from and when in a comment above the package

File: cloudnet-lib/src/main/java/net/md_5/bungee/api/chat/BaseComponent.java [hunk 6]
```diff
@@ -0,0 +1,503 @@
+package net.md_5.bungee.api.chat;
```
Comment (GiantTreeLP) [inline cloudnet-lib/src/main/java/net/md_5/bungee/api/chat/BaseComponent.java:1]: Please mark where this file has come from and when in a comment above the package

File: cloudnet-lib/src/main/java/net/md_5/bungee/api/chat/ClickEvent.java [hunk 7]
```diff
@@ -0,0 +1,54 @@
+package net.md_5.bungee.api.chat;
```
Comment (GiantTreeLP) [inline cloudnet-lib/src/main/java/net/md_5/bungee/api/chat/ClickEvent.java:1]: Please mark where this file has come from and when in a comment above the package

File: cloudnet-lib/src/main/java/net/md_5/bungee/api/chat/ComponentBuilder.java [hunk 8]
```diff
@@ -0,0 +1,388 @@
+package net.md_5.bungee.api.chat;
```
Comment (GiantTreeLP) [inline cloudnet-lib/src/main/java/net/md_5/bungee/api/chat/ComponentBuilder.java:1]: Please mark where this file has come from and when in a comment above the package

File: cloudnet-lib/src/main/java/net/md_5/bungee/api/chat/HoverEvent.java [hunk 9]
```diff
@@ -0,0 +1,24 @@
+package net.md_5.bungee.api.chat;
```
Comment (GiantTreeLP) [inline cloudnet-lib/src/main/java/net/md_5/bungee/api/chat/HoverEvent.java:1]: Please mark where this file has come from and when in a comment above the package

File: cloudnet-lib/src/main/java/net/md_5/bungee/api/chat/KeybindComponent.java [hunk 10]
```diff
@@ -0,0 +1,86 @@
+package net.md_5.bungee.api.chat;
```
Comment (GiantTreeLP) [inline cloudnet-lib/src/main/java/net/md_5/bungee/api/chat/KeybindComponent.java:1]: Please mark where this file has come from and when in a comment above the package

File: cloudnet-lib/src/main/java/net/md_5/bungee/api/chat/Keybinds.java [hunk 11]
```diff
@@ -0,0 +1,51 @@
+package net.md_5.bungee.api.chat;
```
Comment (GiantTreeLP) [inline cloudnet-lib/src/main/java/net/md_5/bungee/api/chat/Keybinds.java:1]: Please mark where this file has come from and when in a comment above the package

File: cloudnet-lib/src/main/java/net/md_5/bungee/api/chat/ScoreComponent.java [hunk 12]
```diff
@@ -0,0 +1,91 @@
+package net.md_5.bungee.api.chat;
```
Comment (GiantTreeLP) [inline cloudnet-lib/src/main/java/net/md_5/bungee/api/chat/ScoreComponent.java:1]: Please mark where this file has come from and when in a comment above the package

File: cloudnet-lib/src/main/java/net/md_5/bungee/api/chat/SelectorComponent.java [hunk 13]
```diff
@@ -0,0 +1,57 @@
+package net.md_5.bungee.api.chat;
```
Comment (GiantTreeLP) [inline cloudnet-lib/src/main/java/net/md_5/bungee/api/chat/SelectorComponent.java:1]: Please mark where this file has come from and when in a comment above the package

File: cloudnet-lib/src/main/java/net/md_5/bungee/api/chat/TextComponent.java [hunk 14]
```diff
@@ -0,0 +1,229 @@
+package net.md_5.bungee.api.chat;
```
Comment (GiantTreeLP) [inline cloudnet-lib/src/main/java/net/md_5/bungee/api/chat/TextComponent.java:1]: Please mark where this file has come from and when in a comment above the package

File: cloudnet-lib/src/main/java/net/md_5/bungee/api/chat/TranslatableComponent.java [hunk 15]
```diff
@@ -0,0 +1,240 @@
+package net.md_5.bungee.api.chat;
```
Comment (GiantTreeLP) [inline cloudnet-lib/src/main/java/net/md_5/bungee/api/chat/TranslatableComponent.java:1]: Please mark where this file has come from and when in a comment above the package

File: cloudnet-lib/src/main/java/net/md_5/bungee/chat/BaseComponentSerializer.java [hunk 16]
```diff
@@ -0,0 +1,152 @@
+package net.md_5.bungee.chat;
```
Comment (GiantTreeLP) [inline cloudnet-lib/src/main/java/net/md_5/bungee/chat/BaseComponentSerializer.java:1]: Please mark where this file has come from and when in a comment above the package

File: cloudnet-lib/src/main/java/net/md_5/bungee/chat/ComponentSerializer.java [hunk 17]
```diff
@@ -0,0 +1,103 @@
+package net.md_5.bungee.chat;
```
Comment (GiantTreeLP) [inline cloudnet-lib/src/main/java/net/md_5/bungee/chat/ComponentSerializer.java:1]: Please mark where this file has come from and when in a comment above the package

File: cloudnet-lib/src/main/java/net/md_5/bungee/chat/KeybindComponentSerializer.java [hunk 18]
```diff
@@ -0,0 +1,29 @@
+package net.md_5.bungee.chat;
```
Comment (GiantTreeLP) [inline cloudnet-lib/src/main/java/net/md_5/bungee/chat/KeybindComponentSerializer.java:1]: Please mark where this file has come from and when in a comment above the package

File: cloudnet-lib/src/main/java/net/md_5/bungee/chat/ScoreComponentSerializer.java [hunk 19]
```diff
@@ -0,0 +1,44 @@
+package net.md_5.bungee.chat;
```
Comment (GiantTreeLP) [inline cloudnet-lib/src/main/java/net/md_5/bungee/chat/ScoreComponentSerializer.java:1]: Please mark where this file has come from and when in a comment above the package

File: cloudnet-lib/src/main/java/net/md_5/bungee/chat/SelectorComponentSerializer.java [hunk 20]
```diff
@@ -0,0 +1,28 @@
+package net.md_5.bungee.chat;
```
Comment (GiantTreeLP) [inline cloudnet-lib/src/main/java/net/md_5/bungee/chat/SelectorComponentSerializer.java:1]: Please mark where this file has come from and when in a comment above the package

File: cloudnet-lib/src/main/java/net/md_5/bungee/chat/TextComponentSerializer.java [hunk 21]
```diff
@@ -0,0 +1,35 @@
+package net.md_5.bungee.chat;
```
Comment (GiantTreeLP) [inline cloudnet-lib/src/main/java/net/md_5/bungee/chat/TextComponentSerializer.java:1]: Please mark where this file has come from and when in a comment above the package

File: cloudnet-lib/src/main/java/net/md_5/bungee/chat/TranslatableComponentSerializer.java [hunk 22]
```diff
@@ -0,0 +1,39 @@
+package net.md_5.bungee.chat;
```
Comment (GiantTreeLP) [inline cloudnet-lib/src/main/java/net/md_5/bungee/chat/TranslatableComponentSerializer.java:1]: Please mark where this file has come from and when in a comment above the package

File: cloudnet-lib/src/main/java/net/md_5/bungee/chat/TranslationRegistry.java [hunk 23]
```diff
@@ -0,0 +1,111 @@
+package net.md_5.bungee.chat;
```
Comment (GiantTreeLP) [inline cloudnet-lib/src/main/java/net/md_5/bungee/chat/TranslationRegistry.java:1]: Please mark where this file has come from and when in a comment above the package

**Review by GiantTreeLP — CHANGES_REQUESTED** [review]
Please remove files that don't belong to this PR, change the target to development and check the copyright on the blatantly copied files.

As this is a bigger PR, please include tests to check whether the serialization is done correctly.

**Comment by GiantTreeLP** [comment]
Closed, please file proper PRs and don't add unrelated code.

**Pull request closed without merging**